packages/localizations/src/es-ES.ts

```diff
@@ -71,6 +71,15 @@ export const esES: LocalizationResource = {
   membershipRole__admin: 'Administrador',
   membershipRole__basicMember: 'Miembro',
   membershipRole__guestMember: 'Invitado',
+  passwordComplexity: {
+    sentencePrefix: '',
```

Comment:
@panteliselef I think you can omit these and they will default to `us` no?

cc: @nikosdouvlis

Reply:
Yup, they will default to the `us` resource - however Id keep these so its apparent that they are not yet localised.

Reply:
Hmm, I've seen using empty strings in other places as well, so I wanted to follow the pattern. If this prevent the US (default) translation to be displayed, I can remove them.